web/styles/settings.css

```diff
@@ -1725,6 +1725,7 @@ label.preferences-radio-choice-label {
     & textarea {
         /* Subtract (1px border and 6px padding) on each side */
         width: calc(var(--modal-input-width) - 14px);
+        max-width: calc(100% - 14px);
```

Comment:
Have done this change in general for all `textarea` elements since we use same class `settings_textarea` for all of them whether they are in settings overlay or in modals.

Also, for a couple of cases of textarea used in modals, we set width as `97%` instead of subtracting the exact padding number from 100% width, so we might want to fix that as well. So, maybe we can do both these things in a separate PR.